### PR TITLE
NEW(Docs) project html documentation

### PR DESCRIPTION
**Pull request by jiridanek**
https://jiridanek.github.io/YamlConfiger/index.html

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# YamlConfiger --- AMQ configurator
+
+This tool can generate a set of configuration files mainly needed for
+AMQ Broker, but it is not limited to only generating files for one product.
+
+YamlConfiger has a user facing Command Line Tool for quick and easy command line usage.
+Furthermore, it is possible to use its API in your python code.
+
+## Documentation
```
Comment (enkeys) [inline docs/index.md:9]: Will be great if you can replace this part with a link to https://rh-messaging-qe.github.io/YamlConfiger/index.html

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+this readme and docstrings, for now. Sorry about that.
+
+I would like to have [readthedocs.org](http://readthedocs.org) documentation.
```
Comment (jiridanek) [inline docs/index.md:13]:
```suggestion
Formatted documentation can be viewed at [rh-messaging-qe.github.io/YamlConfiger/index.html](https://rh-messaging-qe.github.io/YamlConfiger/index.html).
```

File: docs/index.md (hunk 1 above)
Comment (jiridanek) [reply docs/index.md:9]: done in a suggestion

File: docs/install.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Getting started
+
+* python 3.5+ or python2.7
```
Comment (jiridanek) [inline docs/install.md:3]: is 2.7 supported? i noticed it was not listed in the other pr, for tox

Comment (jiridanek) [inline docs/install.md:3]:
```suggestion
* Python 3.6+
```

Comment (jiridanek) [reply docs/install.md:3]: we think it shouldn't be, i'll make the change

File: docs/install.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* python virtualenv recommended (install via system package manager
+or `pip install --user virtualenv`)
+
+for contributors:
+* requirements from requirements.txt (there are Dev and QA requirements as well)
+
+## From git
+
+```bash
+git clone git@bitbucket.org:msgqe/amqcfg.git
```
Comment (michalxo) [inline docs/install.md:14]: github link? Update needed I suppose here

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## Contributing
+
+If you find a bug or room for improvement, submit either a ticket or PR.
+
+## Contributors
+
+_Alphabetically ordered_
+
+* Michal Tóth <mtoth@redhat.com>
```
Comment (michalxo) [inline docs/index.md:21]: Maybe this list should be changed.... Added Dominik, Howard, Jiri already...
AUTHORS.md?

**Review by michalxo — CHANGES_REQUESTED** [review]
Few minor issues. Else looks good to me.

File: docs/index.md (hunk 5 above)
Comment (jiridanek) [reply docs/index.md:21]: Either AUTHORS.md, or we could add CODEOWNERS to the file tree https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners, and https://github.com/rh-messaging-qe/YamlConfiger/graphs/contributors to docs

File: docs/install.md (hunk 4 above)
Comment (jiridanek) [inline docs/install.md:14]:
```suggestion
git clone https://github.com/rh-messaging-qe/YamlConfiger.git
```

Comment (jiridanek) [reply docs/install.md:14]: suggested

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+* Otavio Piske <opiske@redhat.com>
+* Sean Davey <sdavey@redhat.com>
+* Zdenek Kraus <zkraus@redhat.com> (maintainer)
```
Comment (jiridanek) [inline docs/index.md:24]:
```suggestion
## Maintainers

_Alphabetically ordered_

* Michal Tóth <mtoth@redhat.com>
* Dominik Lenoch <dlenoch@redhat.com>
* Zdenek Kraus <zkraus@redhat.com>

See GitHub statistics for an [up-to-date list of contributors](https://github.com/rh-messaging-qe/YamlConfiger/graphs/contributors).
```

File: docs/index.md (hunk 5 above)
Comment (jiridanek) [reply docs/index.md:21]: doc updated in comments, CODEOWNERS is left for further consideration

File: docs/index.md (hunk 6 above)
Comment (michalxo) [reply docs/index.md:24]: I don't think I am a maintainer. Probably just Zdenek and Dominik are :)

Comment (jiridanek) [reply docs/index.md:24]: Ok, I'll delete you.

**Review by michalxo — APPROVED** [review]

**Comment by enkeys** [comment]
It's merged already probably because @michalxo approved it, but the main https://github.com/rh-messaging-qe/YamlConfiger/edit/master/README.md could be updated too for the relevant edits. 

```
# Documentation
Formatted documentation can be viewed at rh-messaging-qe.github.io/YamlConfiger/index.html.
```

**Pull request merged**